tests/test_aiohttp_apistrap_extension.py

```diff
@@ -0,0 +1,152 @@
+import pytest
+import json
+
+from apistrap.errors import ApistrapExtensionError
+from apistrap.aiohttp import AioHTTPApistrap
+from aiohttp import web
+
+
+async def test_aiohttp_apistrap_extension_definition_conflict_1():
+    with pytest.raises(ValueError):
+        oapi = AioHTTPApistrap()
+        oapi.add_response_definition("name", {"foo": "bar"})
+        oapi.add_response_definition("name", {"baz": "bar"})
+
+
+async def test_aiohttp_apistrap_extension_definition_conflict_2():
+    with pytest.raises(ValueError):
+        oapi = AioHTTPApistrap()
+        oapi.add_request_definition("name", {"foo": "bar"})
+        oapi.add_request_definition("name", {"baz": "bar"})
+
+
+async def test_aiohttp_getters():
+    oapi = AioHTTPApistrap()
+    oapi.title = "Title"
+    oapi.description = "Description"
+    assert oapi.title == "Title"
+    assert oapi.description == "Description"
+
+
+async def test_aiohttp_spec_url(aiohttp_client):
+    oapi = AioHTTPApistrap()
+
+    oapi.spec_url = "/myspecurl.json"
+    assert oapi.spec_url == "/myspecurl.json"
+
+    app = web.Application()
+    routes = web.RouteTableDef()
+    oapi.init_app(app)
+
+    @routes.get('/')
```

Comment:
I think you could safely remove the route definition and just check the the spec is present on the configured URL.